### ISSUE TITLE
Readme of 0.4.x branch is incorrect.

### ISSUE THREAD

**Issue by ThatRedox** [chunky-dev/chunky-denoiser#44]
Currently, the README of the 0.4.x branch is the same as master despite the 0.4.x branch not having the changes the readme mentions.

**Comment by leMaik** [comment]
Good call, this probably happened while cherry-picking some changes on top of the old branch. Closed by https://github.com/chunky-dev/chunky-denoiser/commit/a4022fb4c1bc0a38241d491afebdf7ae7850087a
